source/ui/dataman/AMScanView.cpp

```diff
@@ -1106,15 +1164,7 @@ void AMScanViewMultiView::addScan(int si) {
 
 AMScanViewMultiView::~AMScanViewMultiView() {
 
-	/* Not necessary; deleting the plot should delete its children.
- for(int si=0; si<plotSeries_.count(); si++)
-  for(int ci=0; ci<model()->scanAt(si)->dataSourceCount(); ci++)
-   if(plotSeries_[si][ci]) {
- plot_->plot()->removeItem(plotSeries_[si][ci]);
- delete plotSeries_[si][ci];
-   }*/
-
-	delete plotCursor_;
+//	delete plotCursor_;
 	plot_->deleteLater();
```

Comment:
I guess we need to delete `dragZoomerTool_  and wheelZoomerTool_ ` as what you did for `AMScanViewExclusiveView`